### ISSUE TITLE
Is it possible to set Macintosh as the primary startup option when booting?

### ISSUE THREAD

**Issue by ghostplant** [AsahiLinux/m1n1#396]
Is it possible to set Macintosh as the primary startup option when booting?

Currently, I have to always keep pressing "Power" button to select and boot Macintosh.

**Comment by gennan** [comment]
> Is it possible to set Macintosh as the primary startup option when booting?
> 
> Currently, I have to always keep pressing "Power" button to select and boot Macintosh.

@ghostplant You can try changing it in the settings in macOS, 'Settings -> General -> Startup Disk'.

**Comment by ghostplant** [comment]
Yes, it works. I also found another option which is to boot into "Options" then there is also a window to configure default startup options. Thanks!
